### PR TITLE
fenestrated capillary endothelial cells updates 

### PR DESCRIPTION
**Pull request by Caroline-99**
Updated logical and textual defs for peritubular capillary endothelial cells and subclasses 
- added logical defs
- updated the textual defs and included references
- added extra description in the comments section

Updated glomerular endothelial cell defs 
- linked to GO term GO:0036053
- added extra description in the comments section 
- added references

### CODE REVIEW

**Comment by Caroline-99** [comment]
This could be a separate ticket but I think glomerular endothelial cell should be merged into glomerular capillary endothelial cell as they both refer to the exact same cell type as ( same specialized endothelial cells found in the glomerular capillaries of the kidney). And glomerular endothelial cell could be added as a synonym?

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Review by aleixpuigb — COMMENTED** [review]
I would suggest to use 'that is part of' instead of 'found in'.

**Comment by Caroline-99** [comment]
> I would suggest to use 'that is part of' instead of 'found in'.

Done

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -27313,15 +27316,14 @@ SubClassOf(obo:CL_1001111 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000509
 
 # Class: obo:CL_1001123 (kidney outer medulla peritubular capillary cell)
 
+AnnotationAssertion(obo:IAO_0000115 obo:CL_1001123 "An endothelial cell that is part of the peritubular capillaries of the outer renal medulla.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:27319]: the peritubular capillaries should be singular, 'a peritubular capillary'.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -27313,15 +27316,14 @@ SubClassOf(obo:CL_1001111 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000509
...
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1001123 "KUPO:0001066")
 AnnotationAssertion(rdfs:label obo:CL_1001123 "kidney outer medulla peritubular capillary cell")
 EquivalentClasses(obo:CL_1001123 ObjectIntersectionOf(obo:CL_1001033 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0006341)))
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_1001123 obo:CL_1000616)
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_1001123 obo:CL_1001033)
-SubClassOf(obo:CL_1001123 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0006341))
 
 # Class: obo:CL_1001124 (kidney cortex peritubular capillary cell)
 
+AnnotationAssertion(obo:IAO_0000115 obo:CL_1001124 "An endothelial cell that is part of the peritubular capillaries of the renal cortex.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:27326]: Same comment than above

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, see inline comments

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -3070,6 +3070,7 @@ Declaration(Class(obo:CP_0000043))
 Declaration(Class(obo:D96882F1-8709-49AB-BCA9-772A67EA6C33))
 Declaration(Class(obo:GO_0005903))
 Declaration(Class(obo:GO_0031045))
+Declaration(Class(obo:GO_0036053))
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:3073]: Can you also add this term in the GO imports txt file? I will refresh the imports later.

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**